app/styles/partials/footer.scss

```diff
@@ -23,7 +23,7 @@ footer {
 
 @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
   .main-container {
-    margin: 20px 0;
+    margin: $page-top-margin 0;
```

Comment:
@niranjan94 just tested on IE 11 on browser stack here is the screenshot 
![bs_win8 1_ie_11 0](https://user-images.githubusercontent.com/20153241/27551683-0a80f5c4-5ac2-11e7-802d-e3688c8f7866.jpg)
it works fine with high-resolution images on every screen sizes but a little overflow occurs for the images whose aspect ratio is less than 16:9 here is another screenshot 
![bs_win8 1_ie_11 0 2](https://user-images.githubusercontent.com/20153241/27552106-b3fda4de-5ac3-11e7-8d41-c4a7e9e545ee.jpg)